src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.Production.yaml

```diff
@@ -12,6 +12,7 @@ config:
     DRIVE_WEBHOOK_EXPIRATION_MINUTES: 1440
     DRIVE_WEBHOOK_RENEWAL_PERIOD_MINUTES: 720
     FEATURE_ENABLE_ENTERPRISE: "False"
+    FEATURE_ENABLE_EXTERNAL_COURSE_SYNC: "False"
```

Comment:
I assume this overrides the value set in __main__.

Reply:
@mitodl/devops Can confirm. I just followed the pattern for `FEATURE_ENABLE_ENTERPRISE `.

Reply:
This explicitly sets `FEATURE_ENABLE_ENTERPRISE` to `False` for _just_ the production environment. Does that answer the question?